sql/driver/conn.go

```diff
@@ -53,7 +55,18 @@ func (c *conn) Exec(stmt string, args []driver.Value) (driver.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	return driver.RowsAffected(len(result.Rows)), nil
+	switch t := result.GetUnion().(type) {
+	case nil:
+		return nil, nil
+	case *Response_Result_DDL_:
```

Comment:
See #2377; I wanted to use `database/sql/driver.ResultNoRows` which is not equivalent to 0 rows - it returns an error instead.